analysis/deep2_r23_o32/general.py

```diff
@@ -129,41 +130,43 @@ def run_grid_r23_o32_analysis(dataset, n_split=3, y_correction=False,
     # dataset, combine_flux_ascii, binning_avg_asc)
 
     # Verification Table
-    valid_table.make_validation_table(fitspath)
-    bin_valid_file = join(fitspath, filename_dict['bin_valid'])
+    # If dataset is n_bins the validation table does not have to be revised
+    if dataset == 'n_Bins':
+        valid_table.make_validation_table(fitspath, vmin_4363SN=3.5,
+                                          vmin_5007SN=95, vmax_4363sig=0.75,
+                                          rlmin_4363SN=0, rlmax_4363sig=1.4,
+                                          rlmin_5007SN=140)
+        bin_valid_file = join(fitspath, filename_dict['bin_valid'])
+        bin_valid_rev_file = join(fitspath, filename_dict['bin_valid_rev'])
+        bin_valid_tab = asc.read(bin_valid_file)
+        asc.write(bin_valid_tab, bin_valid_rev_file,
+                  format='fixed_width_two_line', overwrite=True)
```

Comment:
Perhaps I'm misunderstanding something but wouldn't it make more sense to still run `compare_by_eye` to ensure it gets the final catalog? Otherwise, if something is suppose to change, this code will override any possible change and you wouldn't even know it.